docs/admin/administration.md

```diff
@@ -709,31 +709,31 @@ Usage:
           --force       Force stop DAOS system members
 ```
 
-The `--ranks` takes a pattern describing rank ranges e.g., 0,5-10,20-100.
+The `--ranks` takes a pattern describing rank ranges, e.g., 0,5-10,20-100.
 The `--rank-hosts` takes a pattern describing host ranges e.g. storagehost[0,5-10],10.8.1.[20-100].
 
-The output table will indicate action and result.
+The output table will indicate the action and result.
 
 While the engines are stopped, the DAOS servers will continue to
 operate and listen on the management network.
 
 !!! warning
     All engines monitor each other and pro-actively exclude unresponsive
-    members. It is critical to properly stop a DAOS system as with dmg in
-    the case of a planned maintenance on all or a majority of the DAOS
+    members. It is critical to stop a DAOS system with dmg in properly in
```

Comment:
It is critical to properly stop a DAOS system with the `dmg system stop` command at the beginning of a planned ...

Reply:
fixed